spiceypy/tests/test_wrapper.py

```diff
@@ -6750,9 +7336,22 @@ def test_unload():
     spice.furnsh(CoreKernels.testMetaKernel)
     # 4 kernels + the meta kernel = 5
     assert spice.ktotal("ALL") == 5
+    # Make list of FURNSHed non-meta-kernels
+    kernelList = [filnam for filnam,filtyp,srcnam,handle in
```

Comment:
for comprehension is too long, make this a normal nested for loop for readability.

Reply:
Switched to this:

    kernelList = []
    for iKernel in range(spice.ktotal("ALL")):
        filnam, filtyp, srcnam, handle = spice.kdata(iKernel, "ALL", 999, 999, 999)
        if filtyp != "META": kernelList.append(filnam)